content/logs/log_collection/_index.md

```diff
@@ -48,20 +48,34 @@ If an integration does not support logs by default, use the custom file configur
+| ------           | -------   | ------                                                                                                                                                                                                                                                                                                                     |
+| `type`           | mandatory | type of log input source (**tcp** / **udp** / **file**)                                                                                                                                                                                                                                                                    |
+| `port`           | mandatory | Set the port to listen log from if `type` is **tcp** or **udp**.                                                                                                                                                                                                                                                           |
+| `path`           | mandatory | Set the path of the file to tail log from if `type` is **file**.                                                                                                                                                                                                                                                           |
+| `service`        | mandatory | Name of the service owning the log. If you instrumented your service with [Datadog APM][12], this should be the same service name.                                                                                                                                                                                         |
+| `source`         | mandatory | Attribute that defines which integration is sending the logs. If the logs do not come from an existing integration then this field may include a custom source name. But we recommend matching this value to the namespace of any related [custom metrics][2] you are collecting, e.g, `myapp` from `myapp.request.count`) |
+| `sourcecategory` | optional  | Multiple value attribute. Can be used to refine the source attribtue. Example: `source:mongodb, sourcecategory:db_slow_logs`                                                                                                                                                                                               |
+| `tags`           | optional  | Add tags to each log collected, [learn more about tagging][13].                                                                                                                                                                                                                                                            |
+
```

Comment:
New parameter have been added for the above new type:

For journald: `include_units` and `exclude_units`: optional, Include or exclude a specific list of journald units

For windows_event: `channel_path`, Mandatory, specify the list of windows event channel to collect.